cypress/integration/executions_spec.js

```diff
@@ -69,5 +70,98 @@ describe('Dashboard Executions Page', () => {
         });
       });
     });
+
+    it('should show a single execution', () => {
+      cy.contains('nav li a', 'Executions').as('executions');
+      cy.get('@executions').should('have.attr', 'href', '#/executions');
+      cy.get('@executions').click();
+
+      cy.url().should('include', 'executions');
+      cy.contains('.heading--xlarge', 'Executions');
+      const executionName = '50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const executionArn = 'arn:aws:states:us-east-1:596205514787:execution:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf:50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const stateMachine = 'arn:aws:states:us-east-1:596205514787:stateMachine:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf';
+      cy.get('table tbody tr td[class=table__main-asset]').within(() => {
+        cy.get(`a[title=${executionName}]`).click({force: true});
+      });
+
+      cy.contains('.heading--large', 'Execution');
+      cy.contains('.heading--medium', 'Visual workflow');
+
+      const startMatch = fullDate('2018-11-12T20:05:10.401Z');
+      const endMatch = fullDate('2018-11-12T20:05:31.536Z');
+
+      cy.get('.status--process')
+      .within(() => {
+        cy.contains('Execution Status:').next().should('have.text', 'Succeeded');
+        cy.contains('Execution Arn:').next().should('have.text', executionArn);
+        cy.contains('State Machine Arn:').next().should('have.text', stateMachine);
+        cy.contains('Started:').next().should('have.text', startMatch);
+        cy.contains('Ended:').next().should('have.text', endMatch);
+      });
+      cy.get('table tbody tr').as('events');
+      cy.get('@events').its('length').should('be.eq', 7);
+
+      const executionStatusFile = `./test/fake-api-fixtures/executions/status/${executionArn}/index.json`;
+      cy.readFile(executionStatusFile).as('executionStatus');
+
+      cy.get('@executionStatus').its('executionHistory').its('events').then((events) => {
+        cy.get('@events').each(($el, index, $list) => {
+          let timestamp = fullDate(events[index].timestamp);
+          cy.wrap($el).children('td').as('columns');
+          cy.get('@columns').its('length').should('be.eq', 4);
+          let idMatch = `"id": ${index + 1},`;
+          let previousIdMatch = `"previousEventId": ${index}`;
+
+          cy.get('@columns').eq(0).should('have.text', (index + 1).toString());
+          cy.get('@columns').eq(2).should('have.text', timestamp);
+          cy.get('@columns').eq(3).contains('More Details').click();
+          cy.get('@columns').eq(3).contains(idMatch);
+          if (index !== 0) {
+            cy.get('@columns').eq(3).contains(previousIdMatch);
+          }
+          cy.get('@columns').eq(3).contains('Less Details').click();
+        });
+      });
+    });
+
+    it('should show logs for a single execution', () => {
+      cy.contains('nav li a', 'Executions').click();
```

Comment:
Having these assertions at the beginning of this test means it is dependent on the result of the previous test putting you on the right page. That could change as we refactor, so the more isolated we can make them, the better.

I would suggest moving these assertions and the check for the `Logs` link into the previous test. And starting this test with a command to visit the logs page:

`cy.visit(`/#/executions/execution/${executionName}/logs`);`